stingray/fourier.py

```diff
@@ -175,20 +175,37 @@ def poisson_level(norm="frac", meanrate=None, n_ph=None):
     raise ValueError(f"Unknown value for norm: {norm}")
 
 
-def normalize_frac(unnorm_power, dt, n_bin, mean_flux):
+def normalize_frac(unnorm_power, dt, n_bin, mean_flux, background_flux=0):
     """Fractional rms normalization.
 
-    .. math::
-        P = \frac{P_{Leahy}}{\mu}
+    ..math::
+        P = \frac{P_{Leahy}}{\mu} = \frac{2T}{N_{ph}^2}P_{unnorm}
+
+    where :math:`\mu` is the mean count rate, :math:`T` is the length of
+    the observation, and :math:`N_{ph}` the number of photons.
+    Alternative formulas found in the literature substitute :math:`T=N\,dt`,
+    :math:`\mu=N_{ph}/T`, which give equivalent results.
+
+    If the background can be estimated, one can calculate the source rms
+    normalized periodogram as
+    ..math::
+        P = P_{Leahy} * \frac{\mu}{(\mu - \beta)^2}
+
+    or
+    ..math::
+        P = \frac{2T}{(N_{ph} - \beta T)^2}P_{unnorm}
 
-    where :math:`\mu` is the mean count rate
+    where :math:`\beta` is the background count rate.
 
     This is also called the Belloni or Miyamoto normalization.
     In this normalization, the periodogram is in units of
     :math:`(rms/mean)^2 Hz^{-1}`, and the squared root of the
     integrated periodogram will give the fractional rms in the
     required frequency range.
 
+    TODO: Check the actual Belloni normalization, which is a factor
+    T off somewhere.
```

Comment:
Is this supposed to be a TODO in here? Should this be an issue rather than a TODO in the code?